tests/test-browser/chrome-test.js

```diff
@@ -0,0 +1,80 @@
+const assert = require('assert');
+const puppeteer = require('puppeteer');
+const inspect = require('util').inspect;
+
+function getUrl() {
+  return process.argv[2];
+}
+
+function testJupyterLabPage(html) {
+  if (inspect(html).indexOf('jupyter-config-data') === -1) {
+    console.error('Error loading JupyterLab page:');
+    console.error(html);
+  }
+}
+
+async function testApplication(page) {
+  const el = await page.waitForSelector('#browserTest', { timeout: 100000 });
+  console.log('Waiting for application to start...');
+  let testError = null;
+
+  try {
+    await page.waitForSelector('.completed');
```

Comment:
What if we don't catch this here and just let the error bubble up? Then we could get rid of some code farther down.